client/src/components/commons/ReviewCard.jsx

```diff
@@ -57,3 +118,31 @@ const ReadMore = ({ text }) => {
     return <p>{text}</p>;
   }
 };
+
+const DisplayComments = ({ comments }) => {
+  if (!comments) return null
+  const [displayComments, setDisplayComments] = useState(false);
+  
+  const toggleComments = () => {
+    setDisplayComments(!displayComments);
+  }
+
+  return (
+    <div>
+      <div onClick={toggleComments}>Show all comments</div>
```

Comment:
Could you possibly utilize more conditional rendering so that when the comments ARE being displayed, the button text changes to 'HIDE all comments'?